configure-helper.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
```

Comment:
maybe you can create a commit only for this, rule are:
- try to have one commit per change 
- commit should not be dependent of a "future commit" 
So sure, this commit would add dead code without your build system, but it won't break.
I prefer sweet small commit over big fat diff.
fat shaming... sorry, I'm not sorry I don't like fat commit, they're ugly, and generally unnecessary

Reply:
You can do this for examples for .mk files. 🗡